### ISSUE TITLE
left shift doesn't work 

### ISSUE THREAD

**Issue by uomar8** [r57zone/DSAdvance#17]
i have a game all buttons works 👍 except SHIFT button when i tried it with the keyboard only left shift works and right shift  not used in the game and with ps4 controller (wireless mode) SHIFT  doesn't work

is it possible instead of one SHIFT to use left shift and right shift in BINDINGS?

**Comment by r57zone** [comment]
Hi, ok, I'll add it in the next version.

**Comment by uomar8** [comment]
> Hi, ok, I'll add it in the next version.

Thank you, much appreciated

**Comment by r57zone** [comment]
Added in the new version
